### PR TITLE
[FSDE-1249] Fix Internal footer links in the HTML Blueprint code

### PR DESCRIPTION
**Pull request by josecarlosgonzalezv**
- [x] Changed some URLs that were bad.

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-136/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://e8719435fc00bfb2eed5f5af3417968c7150104f-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-136/2/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-136/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://dbbe805b5dec45e5ba45083c7726113e28d8d875-chi-pr.enterprise.ctl.io

File: src/website/views/components/footer/_examples.pug [hunk 1]
```diff
@@ -494,55 +494,65 @@ p.-text
...
-                    </li>
-                    <li>
-                      <a href="https://www.lumen.com/en-us/about/legal/acceptable-use-policy.html" target="_blank">Legal Notices</a>
-                    </li>
-                    <li>
-                      <a href="https://www.lumen.com/en-us/about/legal/privacy-notice.html">Privacy</a>
+                        <a
+                            href="https://www.centurylink.com/aboutus/community/community-development/programs-for-customers-with-disabilities.html"
+                            >Accessibility</a
+                        >
```
Comment (janibolkvadze) [inline src/website/views/components/footer/_examples.pug:502]: I this proper indentation / linting?

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-136/4/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-136/5/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-136/6/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://e7debeb5063306a9f1f681e8cbc38756c230c7b6-chi-pr.enterprise.ctl.io

File: src/website/views/components/footer/_examples.pug (hunk 1 above)
Comment (josecarlosgonzalezv) [reply src/website/views/components/footer/_examples.pug]: Solved with 2 spaces.

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-136/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://dba8229312fdf5400f25062641db45441d51cc66-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-136/8/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://8baf27fc4c9f0c97fbb8b0368b2def049ef95f97-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-136/9/. :x:

**Review by jllr — APPROVED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-136/10/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://35e902a305e6fc0428a1fd90b0173b4cc63f4fc5-chi-pr.enterprise.ctl.io

**Pull request merged**